### PR TITLE
sphinx: update of features.rst

### LINKED ISSUE #1371: features.rst did not convert from LaTex
All the actual feature names to be used in myconfig.hpp seem to be missing

### PR DESCRIPTION
**Pull request by KaiSzuttor**
Fixes #1371

Description of changes:
 - added missing parts of the features documentation

### CODE REVIEW

File: doc/sphinx/features.rst [hunk 1]
```diff
@@ -16,126 +16,125 @@ header file:
...
 
--  By default, all coordinates in are periodic. With
-   ``PARTIAL_PERIODIC`` turned on, the global variable ``periodic`` (see
+-  ``PARTIAL_PERIODIC`` By default, all coordinates in |es| are periodic. With
+   ``PARTIAL_PERIODIC`` turned on, the |es| global variable ``periodic`` (see
    section ) controls the periodicity of the individual coordinates.
    Note that this slows the integrator down by around :math:`10-30\%`.
 
--  This switches on the various electrostatics algorithms, such as P3M.
+-  ``ELECTROSTATICS`` This switches on the various electrostatics algorithms, such as P3M.
```
Comment (jonaslandsgesell) [inline doc/sphinx/features.rst]: use "enables the use of" instead of "switches on". e.g. p3m still has to be tuned etc.

**Comment by KaiSzuttor** [comment]
this is done

File: doc/sphinx/inter.rst [hunk 2]
```diff
@@ -1310,7 +1312,7 @@ length, which measures the strength of the electrostatic interaction. As
 a special case, when the thermostat is switched off, the value of
 Bjerrum length you enter is treated as :math:`l_B k_B T` rather than
 :math:`l_B`. This is used to perform an NVE integration (see also
-section :ref:`thermostat`).
+section :ref:`\`\`thermostat\`\`\: Setting up the thermostat`).
```
Comment (rempferg) [inline doc/sphinx/inter.rst:1313]: Why are the backticks escaped?

File: doc/sphinx/setup.rst [hunk 3]
```diff
@@ -73,8 +75,7 @@ or by calling the corresponding ``get_state`` methods like::
     
     gamma_rot = espressomd.System().thermostat.get_state()[0][’gamma_rotation’]
 
-
-.. _thermostat:
+.. _\`\`thermostat\`\`\: Setting up the thermostat:
```
Comment (rempferg) [inline doc/sphinx/setup.rst:78]: Why are the backticks escaped?

File: doc/sphinx/setup.rst [hunk 4]
```diff
@@ -393,7 +394,7 @@ Be aware that this feature is neither properly examined for all systems
 nor is it maintained regularly. If you use it and notice strange
 behaviour, please contribute to solving the problem.
 
-.. _nemd:
+.. _\`\`nemd\`\`\: Setting up non-equilibirum MD:
```
Comment (rempferg) [inline doc/sphinx/setup.rst:399]: Why are the backticks escaped?

**Review by rempferg — CHANGES_REQUESTED** [review]

File: doc/sphinx/features.rst (hunk 1 above)
Comment (rempferg) [reply doc/sphinx/features.rst]: If you are sure something should be changed, you can just do it right here. Click the pencil button on the top right.

File: doc/sphinx/inter.rst (hunk 2 above)
Comment (KaiSzuttor) [reply doc/sphinx/inter.rst:1313]: Because rst interprets backticks.

File: doc/sphinx/setup.rst (hunk 3 above)
Comment (KaiSzuttor) [reply doc/sphinx/setup.rst:78]: Because rst interprets backticks.

File: doc/sphinx/setup.rst (hunk 4 above)
Comment (KaiSzuttor) [reply doc/sphinx/setup.rst:399]: Because rst interprets backticks.

**Pull request merged**